src/UseCase.ts

```diff
@@ -116,7 +116,7 @@ export abstract class UseCase extends Dispatcher implements UseCaseLike {
      *
      *  FIXME: mark this as `abstract` property.
      */
-    execute<R>(..._: Array<any>): R {
+    execute(..._: Array<any>): any {
```

Comment:
I've remove `<R>` from UseCase#execute.
Is it nessary for UseCase?
`UseCase#execute` will be overwritten by sub class.